packages/bpk-stylesheets/index.scss

```diff
@@ -16,65 +16,67 @@
  * limitations under the License.
  */
 
-@import '~normalize.css';
-@import '../bpk-mixins/index.scss';
+@layer base {
```

Comment:
I suggest that before this line we define all the layers upfront, something like: 

`@layer base, components, overrides`;